lib/stat_tracker.rb

```diff
@@ -1,16 +1,15 @@
 class StatTracker
-
-  
-
-
-
-
-
-
-
-
-
-
-
-
+  attr_reader :games,
+              :teams,
+              :game_teams
+
+  def initialize(data_files)
+    @games = data_files[:games]
+    @teams = data_files[:teams]
+    @game_teams = data_files[:game_teams]
+  end
+
+  def self.from_csv(data_files)
```

Comment:
Great job on this method - the name definitely threw me for a loop the first time I saw it!